src/main/features/ccj/routes/check-and-send.ts

```diff
@@ -29,12 +33,15 @@ export default express.Router()
   .post(
     Paths.checkAndSendPage.uri,
     FormValidator.requestHandler(Declaration, Declaration.fromObject),
-    (req: express.Request, res: express.Response, next: express.NextFunction) => {
+    ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
       const form: Form<Declaration> = req.body
+      const user: User = res.locals.user
 
       if (form.hasErrors()) {
         renderView(form, req, res)
       } else {
+        await CCJClient.save(user)
+        await DraftCCJService.delete(res, next)
```

Comment:
`*Client` and `*Service` -- maybe it would be worth to rename one of them for consistency?

Reply:
Not in this PR. it's too large already. But I agree.

Reply:
👍